src/redux/todoReducer.tsx

```diff
@@ -10,6 +11,22 @@ const initialState = {
 //(previousState, action) => newState
 const todoReducer = (state: todoList = initialState, action: any) => {
   switch (action.type) {
+    case "ADD_TODO":
+      const updatedTodos = [...state.tasks];
+      if (action.payload) {
+        const todo = {
+          id: state.tasks.length + 1,
```

Comment:
This can make duplicate ids
You can use https://www.npmjs.com/package/uuid

Reply:
getting some issue in installing plugin, will check with you on monday for this

Reply:
Sure, we will connect for this.